app-ios/App/DroidKaigi2023/DroidKaigi2023.xcodeproj/project.pbxproj

```diff
@@ -490,10 +491,11 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				ASSETCATALOG_COMPILER_GLOBAL_ACCENT_COLOR_NAME = AccentColor;
 				CODE_SIGN_ENTITLEMENTS = DroidKaigi2023/DroidKaigi2023.entitlements;
-				CODE_SIGN_STYLE = Manual;
+				CODE_SIGN_IDENTITY = "Apple Development";
+				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 1;
 				DEVELOPMENT_ASSET_PATHS = "\"DroidKaigi2023/Preview Content\"";
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = DFY27HWN2G;
```

Comment:
I'm not sure if this is ok to merge. 👀

Reply:
Oh, this is not good. I'll remove this 🙏🏼